main/guides/getting-started/README.md

```diff
@@ -9,12 +9,17 @@ After installing the Agoric SDK, you can proceed to [starting a project](./start
 If you're familar with JavaScript development tools such as `node`, `yarn`, and `git`:
 
 ```shell
-node --version # 16.19.1 or higher
+go version
+jq --version
+node --version # 18.16.0 or higher
```

Comment:
**important**:

```suggestion
node --version # LTS version such as 18.16.0
```

"or higher" suggests we will support version 19, 21, etc. but only even numbered node major version are stable. I don't think we'll support version 19, 21, etc.
I think our policy is to support LTS (long term support) releases... though I'm not sure if we've gotten as far as making that a documented policy.

Reply:
I've updated the various :Node version 18.16.0 or later" strings to reflect this LTS versioning instead.